lib/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import {Request} from 'express';
+import * as passport from 'passport';
+
+export interface StrategyOptions {
+  clientID: string;
+  clientSecret: string;
+  callbackURL: string;
+  passReqToCallback?: true;
+  scope?: string[];
+}
+
+export interface StrategyOptionsWithRequest {
+  clientID: string;
+  clientSecret: string;
+  callbackURL: string;
+  passReqToCallback: true;
+  scope?: string[];
+}
+
+export interface VerifyOptions {
+  message: string;
+}
+
+export interface VerifyCallback {
+  (error: any, user?: any, options?: VerifyOptions): void;
+}
+
+export interface VerifyFunctionWithRequest {
+  (req: Request, accessToken: string, refreshToken: string, profile: any, done: VerifyCallback): void;
+}
+
+export interface VerifyFunction {
+  (accessToken: string, refreshToken: string, profile: any, done: VerifyCallback): void;
```

Comment:
You should be able to infer this from above, it's the same but without the request object which we'll see the use of in the below constructors.